python/lsst/drp/tasks/dcr_assemble_coadd.py

```diff
@@ -527,7 +538,7 @@ def run(self, skyInfo, warpRefList, imageScalerList, weightList, supplementaryDa
         ----------
         skyInfo : `lsst.pipe.base.Struct`
             Patch geometry information, from getSkyInfo
-        warpRefList : `list` of `lsst.daf.butler.DeferredDatasetHandle`
+        tempExpRefList : `list` of `lsst.daf.butler.DeferredDatasetHandle`
```

Comment:
Why switch terminology here?  Generally I consider `tempExp` to be the historical name that we're moving away from, and `warp` the newer name we're moving towards.